driver/wnbd_dispatch.c

```diff
@@ -182,8 +182,8 @@ NTSTATUS WnbdDispatchRequest(
                 Element->Srb->SrbStatus = SRB_STATUS_INTERNAL_ERROR;
                 CompleteRequest(Device, Element, TRUE);
                 InterlockedDecrement64(&Device->Stats.UnsubmittedIORequests);
-                WNBD_LOG_ERROR("Could not get SRB %p 0x%llx data buffer. Error: %d.",
-                               Element->Srb, Element->Tag, StorResult);
+                WNBD_LOG_WARN("Could not get SRB %p 0x%llx data buffer. Error: %d.",
```

Comment:
Let's use a separate commit for the log level changes. I think there are quite a few more log messages that will have to be switched to WNBD_LOG_LOUD.

Reply:
Ooops. I've added this one by mistake.

I still need to rebase and split the PR.